src/CommitmentsV2/SFA.DAS.CommitmentsV2/Models/Apprenticeship.cs

```diff
@@ -499,6 +501,16 @@ public DraftApprenticeship CreateCopyForChangeOfParty(ChangeOfPartyRequest chang
                 TrainingCourseOption = this.TrainingCourseOption
             };
 
+            // TODO Should this be limited to CoE
+            if (DeliveryModel == Types.DeliveryModel.PortableFlexiJob)
+            {
+                result.FlexibleEmployment = new FlexibleEmployment
+                {
+                    EmploymentPrice = changeOfPartyRequest.EmploymentPrice.Value,
+                    EmploymentEndDate = changeOfPartyRequest.EmploymentEndDate.Value
+                };
+            }
+
```

Comment:
A Change Of Party, can either be a 'Change Of Employer' or a 'Change Of Provider', I think we should throw an exception if we see an employer trying to do a ChnageOfProvider. It can't work that way

Reply:
This is unfortunate as this branch was based off PA-208.  We don't really want the change of employer code at all yet - that's a separate ticket.  Maybe for now we should just remove these lines completely.

Reply:
I'm okay with that